src/main/java/com/fincatto/documentofiscal/nfe400/classes/NFRetornoStatus.java

```diff
@@ -539,6 +558,8 @@
     CODIGO_999(999, "Rejei\u00e7\u00e3o: Erro n\u00e3o catalogado"),
     CODIGO_9302(9302, "CNPJ Inv\u00e1lido (Zeros, nulo, DV)");
 
+
```

Comment:
?

Reply:
alguma dúvida? se for pelo espaço , deve ter dado um enter sem querer...